biogeophys/FatesPlantHydraulicsMod.F90

```diff
@@ -190,12 +191,12 @@ module FatesPlantHydraulicsMod
        __FILE__
 
 
-  integer, public, parameter :: van_genuchten_type      = 1
-  integer, public, parameter :: campbell_type           = 2
-  integer, public, parameter :: tfs_type                = 3
+  integer, public, parameter :: van_genuchten_type      = 2
+  integer, public, parameter :: campbell_type           = 3
+  integer, public, parameter :: tfs_type                = 1
```

Comment:
also, should we list here which of these are valid for which types of media?

Reply:
I updated the text here.  But the numbers were re-ordered to match the description in the parameter file.  Also, we only support vg and tfs pv/pk hypothesese for plants, campbel could be done but it would require its own parameters and we don't have those in the parameter file.  So as of now, we only use campbel for soil, and it is hard-coded to be used by default there (since the HLMs use it also).